### DISCUSSION TITLE
Does the pandas.Series used have to contain explicit timeseries?

### DISCUSSION THREAD

**Discussion by Lqs66** [georgebv/pyextremes#97] [Q&A]
I noticed that the example provided in the documentation contains explicit time information:
```
Date-Time (GMT),Water Elevation [m NAVD88]
1920-06-01 05:00:00,-0.659
1920-06-01 06:00:00,-0.873
1920-06-01 07:00:00,-1.025
1920-06-01 08:00:00,-0.964
```
Is this necessary, please? When using BM can I specify the size of the block without using the time specification? For example specify 30 numbers as a block.

**Comment by georgebv** [comment]
Your `pandas.Series` index must be datetime/timestamps when working with `pyextremes`. EVA only makes sense in the context of time series. If you don't have absolute time stamps but only know intervals, you can generate timestamps against some imaginary base timestamp (it's irrelevant for EVA).
